steps/reasons-for-appealing/evidence-upload/EvidenceUpload.js

```diff
@@ -191,12 +191,21 @@ class EvidenceUpload extends SaveToDraftStoreAddAnother {
     return (forwardingError, resp, body) => {
       if (!forwardingError) {
         logger.trace('No forwarding error, about to save data', logPath);
-        const b = JSON.parse(body);
-        req.body = {
-          'item.uploadEv': b.documents[0].originalDocumentName,
-          'item.link': b.documents[0]._links.self.href,
-          'item.size': size
-        };
+        try {
+          const b = JSON.parse(body);
+          req.body = {
+            'item.uploadEv': b.documents[0].originalDocumentName,
+            'item.link': b.documents[0]._links.self.href,
+            'item.size': size
+          };
+        } catch (cactchError) {
```

Comment:
I think here error would be only on b.documents[0], try catch doesn't make sense. Here we are just assigning the value not calling any request
if we can just null check on b and document[0] then it will solve our purpose

Reply:
`cactchError` is a spelling mistake

Reply:
If I do not have a try catch, it does not catch exception from tribunals api. It ends the SYA session

Reply:
Modified the logic